.env

```diff
@@ -3,7 +3,7 @@
 #OC_USER=specify_in_.env
 #OC_PASSWORD=specify_in_.env
 #OC_TOKEN=specify_in_.env
-#OC_PROJECT=specify_in_.env
+OC_PROJECT=redhat-ods-applications
```

Comment:
You should not modify the `.env` file, this is a placeholder.

Please revert this change and if you don't want this to be tracked, create a new `.env.local` file and add there the override!

Reply:
dang i left it in